typespec-tests/src/main/java/com/cadl/flatten/FlattenClient.java

```diff
@@ -225,4 +269,28 @@ public void sendLong(SendLongOptions options) {
         }
         sendLongWithResponse(id, request, requestOptions).getValue();
     }
+
+    /**
+     * The update operation.
+     * 
+     * @param id An integer that can be serialized to JSON (`−9007199254740991 (−(2^53 − 1))` to `9007199254740991 (2^53 − 1)` ).
+     * @param request The request parameter.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws HttpResponseException thrown if the request is rejected by server.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @Generated
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public TodoItem update(long id, UpdateRequest request) {
```

Comment:
The idea skipping flatten looks good to me. I'm fine with the current naming as well.